client/src/app/shared/data-sources/project-data-source.spec.ts

```diff
@@ -0,0 +1,19 @@
+import SpyObj = jasmine.SpyObj;
+import {of} from 'rxjs';
+import {IngestService} from '../services/ingest.service';
+import {PaginatedDataSource} from './paginated-data-source';
+import {ProjectDataSource} from './project-data-source';
+
+describe('ProjectDataSource', () => {
+
+  beforeEach(() => {
+  });
+
+  describe('filterByFieldAndValue', () => {
+    it('filterByFieldAndValue', (done) => {
+      const ds = new ProjectDataSource(null);
+      ds.filterByFieldAndValue('x', 'y');
```

Comment:
should there be an assertion here?

Reply:
There will be, I am thinking about what should be tested still. I wanted to get some code out for review.

Reply:
ah, I see that you're excluding it for now.

Reply:
I am still thinking about what to test here.